python/ray/data/datasource/parquet_datasource.py

```diff
@@ -50,22 +50,10 @@
 
 logger = logging.getLogger(__name__)
 
-FRAGMENTS_PER_META_FETCH = 6
-PARALLELIZE_META_FETCH_THRESHOLD = 24
-
 # The `num_cpus` for each metadata prefetching task.
 # Default to 0.5 instead of 1 because it is cheaper than normal read task.
 NUM_CPUS_FOR_META_FETCH_TASK = 0.5
```

Comment:
this one should be moved as well?

Reply:
This is used in the `ParquetDatasource` implementation, and not by the meta provider: https://github.com/ray-project/ray/blob/9f03bbcf9f9c837f832cc081d18f92ede86a3973/python/ray/data/datasource/parquet_datasource.py#L305